web/courses_test.go

```diff
@@ -20,13 +20,13 @@ func TestCreateAndGetCourse(t *testing.T) {
 
 	client, tm, _ := MockClientWithUser(t, db)
 
-	admin := qtest.CreateFakeUser(t, db, 1)
+	admin := qtest.CreateAdminUser(t, db, "fake")
```

Comment:
Note that in another PR, I've remove `CreateAdminUser`, replacing it with `CreateFakeUser`... to reduce the number of "almost identical" user creation helpers. So this would probably cause merge problems down the line...

Reply:
I see. Will "CreateNamedUser" remain? Some tests expect that the user has a non-empty name.

Reply:
Yes, that's the only remaining one in addition to the FakeUser one.

Reply:
Replaced previously changed to `CreateAdminUser` with `CreateNamedUser` to avoid future merge conflicts.